part1/chapter7/quiz1/questions/question_1.rb

```diff
@@ -19,3 +19,34 @@
 # * Note: You can assume the user will only enter integers separated
 #   by commas with no spaces.
 
+def number_to_dash(int)
+  if int == 10
+    puts "----------"
+  elsif int == 9
+    puts "---------"
+  elsif int == 8
+    puts "--------"
+  elsif int == 7
+    puts "-------"
+  elsif int == 6
+    puts "------"
+  elsif int == 5
+    puts "-----"
+  elsif int == 4
+    puts "----"
+  elsif int == 3
+    puts "---"
+  elsif int == 2
+    puts "--"
+  elsif int == 1
+    puts "-"
+  else
+    puts " "
+  end
+end
```

Comment:
This solution works, but it’s rather unwieldy. One day, a user of this program might ask that we extend it so that you can enter values up to 1000. This code isn’t easy to change in that direction: you have to type a new variable for every possible integer the user might enter! There are various ways around that. Ruby lets you multiply strings by integers, or you could use a while loop with a counter. [Good code is easy to change](https://spin.atomicobject.com/2018/12/05/first-principle-of-programming/).